src/Error/ErrorLogger.php

```diff
@@ -55,7 +52,62 @@ public function __construct(array $config = [])
     }
 
     /**
-     * @inheritDoc
+     * Log an error to Cake's Log subsystem
+     *
+     * @param \Cake\Error\PhpError $error The error to log
+     * @param bool $includeTrace Should the log message include a stacktrace
+     * @param ?\Psr\Http\Message\ServerRequestInterface $request The request if in an HTTP context.
+     * @return bool Logging success
```

Comment:
Does the bool return have any practical use? The callers simply ignore the return anyway.

Reply:
That's fair. We can do void